modules/router/templates/router_include.conf.erb

```diff
@@ -26,6 +26,24 @@ access_log /var/log/nginx/lb-access.log timed_combined;
 access_log /var/log/nginx/lb-json.event.access.log json_event;
 error_log  /var/log/nginx/lb-error.log;
 
+# Include the perl module
+perl_modules perl/lib;
+
+# Function to lowercase a uri
+perl_set $uri_lowercase 'sub {
+        my $r = shift;
+        my $uri = $r->uri;
+        $uri = lc($uri);
+        return $uri;
+}';
+
+# If slug contains no lowercase letters then lowercase it
+# eg www.gov.uk/GOVERNMENT/GUIDANCE -> www.gov.uk/government/guidance
+# eg WWW.GOV.UK/GOVERNMENT/GUIDANCE -> www.gov.uk/government/guidance
+location ~ ^([A-Z]|\W|\d)+$ {
+  rewrite ^(.*)$ $scheme://$host:8080$uri_lowercase;
```

Comment:
Thinking about it, we should probably redirect with a 301 to help our Google juice